.github/workflows/frontend-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: Frontend tests
+on:
+  pull_request
+
+jobs:
+  php:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
+    name: "PHP: Nextcloud ${{ matrix.nextcloud }} - PHP ${{ matrix.php-versions }} - DB ${{ matrix.database }}"
+    strategy:
+      matrix:
+        php-versions: ['7.4']
+        nextcloud: ['20']
+        database: ['sqlite']
+        experimental: [false]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+
+      ### Back to normal setup
+      - name: Set up server non MySQL
+        if: matrix.database != 'mysql'
```

Comment:
It'll never be mysql
```suggestion
```

Reply:
hehe yea that's copy pasted from the php test